db/migrate/20190819121620_create_restaurants.rb

```diff
@@ -0,0 +1,17 @@
+class CreateRestaurants < ActiveRecord::Migration[5.2]
+  def up
+    create_table :restaurants do |t|
+      t.column "name", :string, :limit => 40
+      t.column "cuisine", :string, :limit => 20
+      t.column "rating", :float
```

Comment:
What are the upsides and what are the downsides of keeping this column? Do you intend it to be computed on each restaurant review?

Reply:
The upsides are that each time a restaurant is being requested from the client, the field will not be computed from scratch (and the reviews of the restaurant will not be selected for each READ action). Generally, I think that we expect for a higher number of READ actions of restaurants rather than CREATE or UPDATE actions or reviews.
The downside is that by computing a restaurant's rating each time a review of it is being created or updated, we create high coupling between the 2 models.
I think that because reviews are highly related to restaurants - I should (and will 🙂) compute this field on each  rstaurant review.

Reply:
Define it non-persistent field, calculate it every time we read restaurants